pom.xml

```diff
@@ -1005,6 +1005,97 @@
             <plugin>
                 <artifactId>maven-failsafe-plugin</artifactId>
             </plugin>
+            <plugin>
+                <groupId>org.jacoco</groupId>
+                <artifactId>jacoco-maven-plugin</artifactId>
+                <version>0.8.8</version>
+                <configuration>
+                    <excludes>
+                        <exclude>**/groovy/**</exclude>
```

Comment:
too many excludes, maybe we could remove the other exclude only keep `**/groovy/**`

Reply:
Got it. Will remove all other items in the `/groovy/`directory. However it has some other directory like ` <exclude>**/groovyjarjarantlr4/runtime/**</exclude>
                        <exclude>**/groovyjarjarantlr4/runtime/debug/**</exclude>` Do I need to change those exclude like above to `**/groovyjarjarantlr4/**``**/groovyjarjarasm/**` or `**/groovy/**` will exclude all directory which contains `groovy`

Reply:
**/groovyjarjarantlr4/**``**/groovyjarjarasm/**`, is it possible to use **/groovy**/**` to exlude these packages ?

Reply:
I think it probably not in the same package. THus We have to add different package name to exclude them.